docs/rfcs/014-born_digital_workflow/README.md

```diff
@@ -0,0 +1,176 @@
+
+### Cataloguing
+
+Once an initial appraisal has been completed, the files in scope for archiving need to be arranged into archive items and catalogued in Calm. Items should be created in Calm to represent each related set of files, which will be submitted and stored as a single package in the storage service.
+
+This means that an item in Calm corresponds to a bag in the storage service, not an individual file. File-level description is fully automated and the metadata generated is stored in a METS file created by the Archivematica processing workflow.
+
+| Step                   | Description |
+|------------------------|-------------|
+| Find/create collection | Find, or create if necessary, the collection in Calm that the items will be part of. |
+| Find/create series     | Find, or create if necessary, the series and sub-series within the collection that the items will be part of. |
+| Create items           | For each set of related files, create an item in Calm that represents and describes them collectively in the catalogue. This should describe the files as a set, not list details of individual files. |
+
+### Transfer
+
+Once each set of related files as been identified, they need to be assembled into a package that can be transferred into Archivematica for processing and storage. This package is a simple zip file, with the addition of a minimal set of descriptive metadata.
+
+This metadata is used for identification only. The canonical version of the metadata, which is also the data that will be displayed to end users, remains the data in Calm. Similarly, whilst Archivematica does have support for rights information, we will not be supplying this through the workflow. Calm will be the sole location for rights information.
+
+The package created for transfer into Archivematica should reflect the original order of the files, as this structure will be maintained in the storage service and in the file-level metadata presented to users.
+
+The zip file should have the following structure:
+
+	PPABC_1234.zip
+	|-- file1.txt
+	|-- file2.doc
+	|-- file3.jpg
+	|-- folder1
+	    \-- [...] 
+	|-- folder2
+	    \-- [...] 
+	|-- [...]
+	\-- metadata
+	    \-- metadata.csv
+
+ Note the addition of a `metadata` directory, with a single file named `metadata.csv`, which contains item level metadata from Calm. `metadata.csv` should have the following structure:
+
+	filename,dc.title,dc.creator,dc.date,dc.identifier
+	objects,<Title>,<CreatorName>,<Date>,<RefNo>
+
+Once the zip files have been created, they need to be uploaded to AWS S3. As soon as the upload of a zip completes, it should be picked up by Archivematica and a processing workflow started automatically.
+
+Access to S3 requires the installation of a client application. Until this can be made available more widely, zip files should be passed to the Digital Production team for upload to S3.
```

Comment:
What would this involve? The movement/copying of files concerns me. Through doing the digital audit it has become clear that this is how we get ourselves into a pickle. Staff changes and inconsistent practise certainly haven't helped, but there are files all over the place and it's difficult to know to which stage in the workflow they have reached.

Reply:
I think there's a missing transfer to Wellcome step anyway. With the corporate move to Sharepoint, I am not sure how much longer we will have access to a network share as a staging area, plus there are security and tracking issues associated with multiple movements of files, as Jenny points out. 

My understanding was that Archivematica enables iterative processing, ie that we could submit accession bundles directly to S3 and then recall them later for detailed appraisal and cataloguing? There could well be a time lag of several years between the acquisition of digital material and the point of cataloguing, and I'd prefer to have bags stored in S3 during this time than on an internal network drive.

Reply:
There's a backlog feature on Archivematica that I think we could use for this. If we transfer the items into Archivematica, it could sit in the backlog (I believe we can point the backlog storage to be anywhere we want so it could be on S3) anyone can pick them up out of the backlog when they're ready and take them into appraisal/create a SIP for it.